### PR TITLE
Feature/set up repo

### PR DESCRIPTION
**Pull request by JamesGardiner**
Adds some basic repo requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+
+```Shell
+    $ pip install git+git://github.com/ONSDigital/sdc-rabbit-python.git@master
+```
+
+For production deployments a tag should be referenced, rather than master.
+
+### Basic Use
+
+Assuming you are running a virtual environment:
```
Comment (insacuri) [inline README.md]: Super minor but shouldn't it be 'Assuming you are running in a virtual environment'

File: .codecov.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+codecov:
+  branch: master
```
Comment (insacuri) [inline .codecov.yml]: I'll need to double check but I think we can get away without this.  Originally we needed it to set the default branch to develop, but I think without this file, it defaults to master regardless

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+__pycache__
+logs/
+*.pyc
+.Python
+pip-selfcheck.json
+lib/
+bin/
+include/
+coverage.xml
+.coverage
```
Comment (warrenbailey) [inline .gitignore]: Any chance we can add .idea for us lowly Pycharm users

# Pycharm IDE
.idea

File: .codecov.yml (hunk 2 above)
Comment (JamesGardiner) [inline .codecov.yml]: Ok, I'll leave it in for now. We can remove at a later date if possible.

File: .gitignore (hunk 3 above)
Comment (JamesGardiner) [inline .gitignore]: :see_no_evil:

**Pull request merged**